### DISCUSSION TITLE
Compatibility with Cloud based solutions e.g. Azure

### DISCUSSION THREAD

**Discussion by jmonaniat** [sceneview/sceneform-android#211] [Q&A] [answered]
Hello,

This more about query than bug. Is SceneformMaintained compatible with Azure Cloud Spatial Anchor? Is any of the current client solutions/apps using Azure or any other cloud based solution along with SceneformMaintained sdk/lib? If so, can you recommend sample or pointers which can refer to? Also is it possible to replace old SceneForm based solution using SceneformMaintained? Do you have any migration guide or sample for the reference? 

Thanks in advance.

Best
Jvm

**Comment by ThomasGorisse** [comment]
Hi,
For the Azure part, sorry but I have totally no idea of any works with ARCore on it.

For moving to SceneformMaintained from the old Sceneform, if you come from the last 1.16 (glTF, not sfb assets) you will mainly have nothing to change in your code .

**Comment by doranteseduardo** [comment, marked as answer]
Hello,
The Azure Spatial Anchors is nothing but an alternative to the Google Cloud Anchors API.

It must work, yes.
Remember that the CloudSpatialAnchorSession is a wrapper for Arcore Session, same with CloudSpatialAnchor.

**Comment by jmonaniat** [comment]
I believe it was crashing when launching AR mode annotation on `so` library inside sdk/lib.
